html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input type="email" name="email" id="email" required>
+      </div>
+      <div>
+        <label for="select">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="reason">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="language">Favorite Language (choose one)</label><br />
+        <input type="radio" name="language" value="html" /> HTML
+        <input type="radio" name="language" value="css" /> CSS
+        <input type="radio" name="language" value="javascript" /> Javascript
+      </div>
+      <div>
+        <label for="reply">Would you like to sign up?</label><br />
+        <input type="checkbox" name="reply" value="yes" /> Sign me up!
+      </div>
+      <div>
+        <label for="comments">Comments</label><br />
+        <textarea name="comments" cols="20" rows="10"></textarea> <br />
```

Comment:
make sure that the inputs have an `id` attribute which matches the `for` attribute on the labels